docker/Dockerfile.debian-buster-aarch64

```diff
@@ -0,0 +1,28 @@
+# Built from https://github.com/qwandor/cross/blob/context/docker/Dockerfile.context
+FROM ghcr.io/qwandor/cross-context:latest as context
```

Comment:
maybe tag this as a particular version (whatever the current version or cross is?) to avoid the problem that I caused when I updated my branch and rebuilt?

Reply:
Done.